packages/binding-coap/test/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
     "extends": "../tsconfig.json",
     "compilerOptions": {
-        "rootDir": ".."
+        "rootDir": "..",
+        "experimentalDecorators": true
     },
-    "include": ["*.ts", "**/*.ts", "../src/**/*.ts"]
+    "include": ["test/*.ts", "test/**/*.ts"]
```

Comment:
Where are these changes coming from? are related to issue #599?

Reply:
This change doesn't belong here! let me remove it